src/objective-c/GRPCClient/GRPCTypes.h

```diff
@@ -23,7 +23,7 @@
  * Note that a few of these are never produced by the gRPC libraries, but are of
  * general utility for server applications to produce.
  */
-typedef NS_ENUM(NSUInteger, GRPCErrorCode) {
+typedef NS_ENUM(int32_t, GRPCErrorCode) {
```

Comment:
Thanks Daniel! LGTM in general :)

2 thoughts:

Is this a non-breaking change for all clients?
There are a few other enums in this file, as well as throughout the codebase, which i wonder would benefit from the same improvement?

Thanks :)

Reply:
thanks, this change is non-breaking.  This PR changes just the GRPCErrorCode enum. We will follow up with incremental changes to the rest of the enum when we made corresponding g3 patches.

Reply:
Ok thanks :)

(In general, for the sake of API consistency i'd prefer our APIs to use similar types if possible.. But it's fine with me since you're going to do a bunch of follow-ups soon :))